orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/image/ImageFinder.java

```diff
@@ -28,5 +29,14 @@
   interface ImageDetails {
     String getImageId();
     String getImageName();
+    JenkinsDetails getJenkins();
+  }
+
+  class JenkinsDetails extends HashMap<String, String> {
```

Comment:
The global context has different serialization requirements but making it a Map skirts all of that.